server/db.js

```diff
@@ -246,6 +254,10 @@ var updateUserById = function(userId, info, callback){
         update.$set.email = info.email;
     }
 
+    if (typeof info.active !== 'undefined') {
```

Comment:
why is `undefined` in quotes?

Reply:
the type is returned as a string

Reply:
I cant user !info.active, cuz if active is false, it will skip setting the active state. So i have to check if it is defined. But typeof returns a string